packages/generators/init-template/default/package.json.js

```diff
@@ -1,6 +1,9 @@
 module.exports = (isUsingDevServer) => {
     const scripts = {
         build: 'webpack --mode=production',
+        'build:dev': 'webpack',
```

Comment:
```suggestion
        'build:dev': 'webpack --mode=development',
```

Reply:
The generated webpack config has the mode set to development.

https://github.com/webpack/webpack-cli/blob/6fb1215c8ac82e88ee85861e987b476af47acc61/packages/generators/init-template/default/webpack.configjs.tpl#L6

Reply:
I think it will be much clear if we mention `--mode=development` though both are the same.
Thoughts /cc @webpack/cli-team

Reply:
yeah flag is more explicit than implied via config